### PR TITLE
Guide user if brouter is not installed (fix #9819)

### LINKED ISSUE #9819: Deactive routing settings (or guide user) if brouter is not installed
Since we have the quick settings out on the market, it seems many users like it and use it.
Anyway since then also some users complaint came in like "I selected bike but it still shows the straight line. Why is that?". It turned out those users did not have brouter installed.

Indeed those settings are shown despite brouter being not installed on the device.
The easy solution would be to hide these settings completely in that case. The better solution would be to grey out all options except "straight line" and when the user clicks on another option to inform him, that he needs brouter to use that feature (ideally with link to the app and/or to the user guide).

### PR DESCRIPTION
**Pull request by fm-sys**
Guide the user to the playstore if brouter is not yet installed. For screenshots see https://github.com/cgeo/cgeo/issues/9819#issuecomment-762661620

### CODE REVIEW

File: main/res/values/strings.xml [hunk 1]
```diff
@@ -1287,6 +1287,7 @@
     <string name="map_as_list">Show as list</string>
     <string name="map_select_multiple_items">Select item</string>
     <string name="map_routing">Routing</string>
+    <string name="map_routing_brouter">Routing is only available when <a href="">brouter</a> is installed</string>
```
Comment (fm-sys) [inline main/res/values/strings.xml:1290]: I'm not that good at English, is "when" correct or should it rather be "if"?

Comment (Lineflyer) [reply main/res/values/strings.xml:1290]: yes, IMHO it should be "if"

File: main/res/values/strings.xml [hunk 2]
```diff
@@ -1287,6 +1287,7 @@
     <string name="map_as_list">Show as list</string>
     <string name="map_select_multiple_items">Select item</string>
     <string name="map_routing">Routing</string>
+    <string name="map_routing_brouter">Routing is only available if <a href="">brouter</a> is installed</string>
```
Comment (moving-bits) [inline main/res/values/strings.xml:1290]: LGTM, thanks for implementation. Just a tiny thing: The app is named "BRouter", or "BRouter Offline Navigation" (though I would stay with the short form) - can you adapt this, please?

Comment (fm-sys) [reply main/res/values/strings.xml]: Done!

**Pull request merged**